akkeeper/src/main/scala/akkeeper/config/RichConfig.scala

```diff
@@ -15,102 +15,133 @@
  */
 package akkeeper.config
 
-import akkeeper.api.DeployContainer
+import java.time.{Duration => JavaDuration}
+import java.util.concurrent.TimeUnit
+
+import akka.util.Timeout
 import akkeeper.common.ContainerDefinition
 import akkeeper.master.service.MasterService
 import akkeeper.storage.zookeeper.ZookeeperClientConfig
 import com.typesafe.config.{Config, ConfigValueFactory}
 import org.apache.hadoop.conf.Configuration
 import org.apache.hadoop.fs.{FileSystem, Path}
+
 import scala.collection.JavaConverters._
+import scala.collection.mutable
+import scala.concurrent.duration._
 
 private[akkeeper] final class RichConfig(config: Config) {
 
-  def withMasterRole: Config = {
-    config.withValue("akka.cluster.roles",
-      ConfigValueFactory.fromIterable(Seq(MasterService.MasterServiceName).asJava))
+  implicit def javaDuration2ScalaDuration(value: JavaDuration): FiniteDuration = {
```

Comment:
[Readability suggestion] Let's move it to the end of this class definition, since this is a utility method and doesn't really bring any useful information for someone who's reading this code.